src/SingleProject/Resizetizer/src/AndroidAdaptiveIconGenerator.cs

```diff
@@ -64,8 +64,18 @@ void ProcessBackground(List<ResizedImageInfo> results, DirectoryInfo fullInterme
 			{
 				var dir = Path.Combine(fullIntermediateOutputPath.FullName, dpi.Path);
 				var destination = Path.Combine(dir, backgroundDestFilename);
+				var destinationExists = File.Exists(destination);
 				Directory.CreateDirectory(dir);
 
+				var sourceModifiedDateTime = backgroundExists ? File.GetLastWriteTimeUtc(backgroundFile) : System.DateTime.MinValue;
+				var destinationModifiedDateTime = destinationExists ? File.GetLastWriteTimeUtc(destination) : System.DateTime.MinValue;
```

Comment:
A bit extra and requires more code changes, but I am wondering if this can be a helper call like this:

```cs
(bool Exists, DateTime Modified) Exists(string path)
{
    var exists = File.Exists(path);
    var modified = exists ? File.GetLastWriteTimeUtc(path) : System.DateTime.MinValue;
    return (exists, modified);
}
```

I am worried that we are gonna mix a filename because copy-paste is easy to break. And also, 1 line with 2 values.

Reply:
@mattleibow 

A question. In XA we have a rule where if its not obvious what the return type of a method is we use the actual type not `var`.  

So in this case which would maui prefer given I'm adding the new method to `Utils`.

```
(bool Exists, DateTime Modified) sourceExists = Utils.FileExists(Info.Filename);
```

or

```
var sourceExists = Utils.FileExists(Info.Filename);
```

Reply:
I would go for the first option as that also keeps the same variables names and potentially reducing code diff. But also, I would expand the tuples in most cases - unless there are like 4+ items.

Reply:
Isn't there also a 

```cs
var (exists, dateModified) = Utils.Xxx();
```

Reply:
That last one is pretty cool. I'll use that 👍